### DISCUSSION TITLE
API Downloads and AUP?

### DISCUSSION THREAD

**Discussion by GiraffeDog** [Dineshkarthik/telegram_media_downloader#127] [Q&A] [answered]
Is there anything we need to be concerned with about over using these tools to download from Telegram groups/conversations? I'm looking at using it to download audio files uploaded to a group, but don't want them to be blocked/affected by my bulk downloads.

**Comment by Dineshkarthik** [comment, marked as answer]
@GiraffeDog the download script is not going to affect anything, it is the automated equivalent of you manually pressing the download and save in your telegram app.

**Comment by GiraffeDog** [comment]
Thanks - its clunking away now, with no issues :)

Really impressed with the speed of access via API! Thanks for building this!
